src/components/Button/Button.scss

```diff
@@ -334,7 +334,7 @@ $button-commandButtonHeight: 40px;
   }
 }
 
-.ms-Button.ms-Button--icon {
+.ms-Button--icon {
```

Comment:
i definitely think this shouldn't be changed. We fixed the teachingbubble button style to be more specific.